pkg/build/build.go

```diff
@@ -85,10 +93,16 @@ func (b *Builder) prepareBuildDir() error {
 		b.eibBuildDir = b.buildConfig.BuildDir
 	}
 	b.combustionDir = filepath.Join(b.eibBuildDir, "combustion")
+	b.rpmBuildDir = filepath.Join(b.combustionDir, "rpms")
```

Comment:
Why not the same directory like: https://documentation.suse.com/sle-micro/5.5/html/SLE-Micro-all/cha-images-combustion.html

Reply:
My gut reaction was that everything would be in a flat combustion directory. That might get a bit messy for debugging large builds, but I think we should start with that model and only make it more complex if we find a need to. So +1 to Alberto's comment.

Reply:
sounds good to me, my initial thinking was just maintaining the organization from the config dir which I commented on in another reply, but I'll make this change